code/drasil-code/Language/Drasil/Code/ExtLibImport.hs

```diff
@@ -4,12 +4,13 @@
 module Language.Drasil.Code.ExtLibImport (ExtLibState(..), auxMods, defs, 
   imports, modExports, steps, genExternalLibraryCall) where
 
-import Language.Drasil
+import Language.Drasil (HasSpace(typ), getActorName, NamedArgument)
```

Comment:
It surprises me that `getActorName` is exported by `Language.Drasil`. Wouldn't this be move to `drasil-code`?

Reply:
`getActorName` is another function like `getInnerSpace` -- exact same functionality but instead of it being for the "Vect" Space, it's for the "Actor" Space. I guess, if anything, it would be another function marked for removal, like `getInnerSpace`.